libs/core/langchain_core/runnables/passthrough.py

```diff
@@ -611,6 +611,28 @@ async def input_aiter() -> AsyncIterator[Dict[str, Any]]:
 class RunnablePick(RunnableSerializable[Dict[str, Any], Dict[str, Any]]):
     """
     Runnable that picks keys from Dict[str, Any] inputs.
+
+    RunnablePick class represents a runnable that selectively picks keys from a dictionary input. 
+    It allows you to specify one or more keys to extract from the input dictionary.
+    It returns a new dictionary containing only the selected keys.
+
+    Example : 
+    .. code-block:: python
```

Comment:
..code-block and the contents needs to be further indented with additional tab -- you can use the other code-block examples as reference